web/portal/user/src/entities/Fax/Fax.tsx

```diff
@@ -0,0 +1,93 @@
+import defaultEntityBehavior, {
+  ChildDecorator as DefaultChildDecorator,
+} from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import EntityInterface, {
+  ChildDecoratorType,
+} from '@irontec/ivoz-ui/entities/EntityInterface';
+import _ from '@irontec/ivoz-ui/services/translations/translate';
+import FaxIcon from '@mui/icons-material/Fax';
+
+import { FaxProperties, FaxPropertyList } from './FaxProperties';
+
+const ChildDecorator: ChildDecoratorType = (props) => {
+  const { routeMapItem } = props;
+
+  const Fax = routeMapItem.entity;
+
+  const isDeletePath = routeMapItem.route === `${Fax.path}/:id`;
+  const isUpdatePath = routeMapItem.route === `${Fax.path}/:id/update`;
+  const isDetailedPath = routeMapItem.route === `${Fax.path}/:id/detailed`;
+
+  if (isUpdatePath || isDeletePath || isDetailedPath) {
+    return null;
+  }
```

Comment:
Have tried to disable acls?

Reply:
Yes, both in EntitiyMaps.ts and in Fax.tsx